bin/sem-apply

```diff
@@ -8,24 +8,33 @@
 #  sem-apply --host <database host> --user <db user> --name <db name>
 #
 # == Examples
-#  sem-apply --url postgresql://postgres@localhost/sample
+#  sem-apply --url postgresql://postgres@localhost:port/sample
 #  sem-apply --host localhost --user web --name test
 #
+require 'tempfile'
 
 load File.join(File.dirname(__FILE__), 'sem-config')
 
 args = SchemaEvolutionManager::Args.from_stdin(:optional => %w(url host port name user dry_run))
-
 db = SchemaEvolutionManager::Db.from_args(args)
-db.bootstrap!
-
 dry_run = args.dry_run.nil? ? false : args.dry_run
+pgpass = args.pgpass.nil? ? false : args.pgpass
+pgpassfile = Tempfile.new('pgpass')
```

Comment:
sem has a Library.with_temp_file helper that we should use here instead of tempfile